source/3d-secure-2.html.erb

```diff
@@ -0,0 +1,132 @@
+        <li>any transactions where either the merchant or payer is outside the EEA</li>
+        <li>merchant initiated payments, for example subscriptions</li>
+        <li>telephone payments</li>
+      </ul>
+
+      <p class="govuk-body">Some payments may be exempted from 3DS2:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>low value transactions, for example for those under £30</li>
+        <li>low risk transactions when the Payment Service Provider (PSP) has low fraud levels across all its platform</li>
+        <li>corporate payments (unless the corporate card is in the name of an individual)</li>
+      </ul>
+
+      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP).</p>
+
+      <h2 class="govuk-heading-m">What 3D Secure 2 means for services</h2>
+      <p class="govuk-body">You may see a reduction in fraud on your service because users need to provide additional authentication before making a payment.</p>
+      <p class="govuk-body">Services are not liable for fraudulent or unrecognised transactions if they are processed by 3DS2. If a user disputes a 3DS2 transaction (for example via a chargeback) on the basis that it is fraudulent or unrecognised, it is the responsibility of the bank to refund the customer if required, because the bank approved 3DS2 transactions.</p>
+      <p class="govuk-body">Services are still liable for transactions that are exempt from 3DS2 and are responsible if a user disputes a transaction on the basis that they did not receive the goods or service they paid for.</p>
+
+      <h2 class="govuk-heading-m">How to set up 3D Secure 2</h2>
+      <p class="govuk-body">The way you set up 3DS2 will depend on your Payment Service Provider (PSP).</p>
+
+      <h3 class="govuk-heading-s">Stripe</h3>
+      <p class="govuk-body">GOV.UK Pay is managing the 3DS2 integration with Stripe so you do not need to make any changes to your integration with us. Stripe will automatically apply exemptions.</p>
+
+      <h3 class="govuk-heading-s">Worldpay</h3>
+      <p class="govuk-body">Worldpay are implementing 3DS2 as part of their Flex product.. You will receive 3DS Flex credentials from Worldpay and <a class="govuk-link" href="https://docs.payments.service.gov.uk/set_up_3dsecure/#set-up-3ds2-3ds-flex">you will need to add them into your Pay service.</a></p>
+      <p class="govuk-body">If you are a Government Banking customer, Government Banking will contact you to let you know when to enable 3DS Flex on your account.</p>
+      <p class="govuk-body">If you have a direct contract with Worldpay (not via Government Banking), Worldpay will send you your 3DS credentials.</p>
+      <p class="govuk-body">Worldpay will send all transactions via 3DS2 or 3DS. Exemptions (see above) will not be applied. If you’d like to apply for exemptions for your transactions, <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact GOV.UK Pay.</a></p>
```

Comment:
Recommend rephrasing to say what is doing the application of exemptions, and that this is saying that exemptions are not automatically applied. For example: "Worldpay does not automatically apply exemptions."

Also, try to avoid the words "above" and "below" as this is not very accessible to people who use screenreaders or other assistive technology